mungegithub/submit-queue/deployment/community/configmap.yaml

```diff
@@ -6,6 +6,7 @@ pr-mungers: size, submit-queue,needs-rebase, sig-mention-handler,
 state: open
 token-file: /etc/secret-volume/token
 period: 5m
+pr-max-wait-time: 2h
```

Comment:
If the default value is desired then the option should probably be left out from the config to avoid cluttering it. I left some default values in the config for important options where it is useful to explicitly indicate the value we are using, but for something like this I think its best to include the option only if a non-default value is desired.